velox/expression/FunctionSignature.h

```diff
@@ -171,7 +178,6 @@ class FunctionSignature {
   // Return a string of the list of argument types.
   std::string argumentsToString() const;
 
- private:
```

Comment:
Accidental change?

Reply:
`AggregateFunctionsSignature` needs to get the `variables_`, so I changed it to `protected`.